pkg/fleet/installer/setup/install.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Does it make sense this file to be owned by @DataDog/fleet?

Reply:
Today yes, in the future we might change it